owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/MainFileListFragment.kt

```diff
@@ -1158,12 +1151,8 @@ class MainFileListFragment : Fragment(),
             }
 
             R.id.action_remove_file -> {
-                if (checkedFiles.size == 1 && !checkedFiles[0].isFolder) {
-                    showRemoveCustomDialog(checkedFiles[0], requireContext())
-                } else {
-                    filesToRemove = checkedFiles
-                    fileOperationsViewModel.showRemoveDialog(filesToRemove)
-                }
+                filesToRemove = checkedFiles
+                fileOperationsViewModel.showRemoveDialog(filesToRemove)
```

Comment:
It can be used `checkedFiles` on `showRemoveDialog` instead of having a variable called `filesToRemove`

Reply:
Ok, we could remove it.